keyboards/treasure/type30/hotswap/keyboard.json

```
@@ -0,0 +1,120 @@
+            { "matrix": [0, 0],"flags": 4, "x": 0, "y": 0 },
+            { "matrix": [0, 1],"flags": 4, "x": 10, "y": 0 },
+            { "matrix": [0, 2],"flags": 4, "x": 20, "y": 0 },
+            { "matrix": [0, 3],"flags": 4, "x": 30, "y": 0 },
+            { "matrix": [0, 4],"flags": 4, "x": 40, "y": 0 },
+            { "matrix": [0, 5],"flags": 4, "x": 50, "y": 0 },
+            { "matrix": [0, 6],"flags": 4, "x": 60, "y": 0 },
+            { "matrix": [0, 7],"flags": 4, "x": 70, "y": 0 },
+            { "matrix": [0, 8],"flags": 4, "x": 80, "y": 0 },
+            { "matrix": [0, 9],"flags": 4, "x": 90, "y": 0 },
+            { "matrix": [0, 10],"flags": 4, "x": 100, "y": 0 },
+            { "matrix": [0, 11],"flags": 4, "x": 110, "y": 0 }
+        ],
+        "max_brightness": 180,
+        "sleep": true
+    },
+    "ws2812": {
+        "pin": "B4"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x12": {
+            "layout": [
+                { "matrix": [0, 0], "x": 0, "y": 0 },
+                { "matrix": [0, 1], "x": 1, "y": 0 },
+                { "matrix": [0, 2], "x": 2, "y": 0 },
+                { "matrix": [0, 3], "x": 3, "y": 0 },
+                { "matrix": [0, 4], "x": 4, "y": 0 },
+                { "matrix": [0, 5], "x": 5, "y": 0 },
+                { "matrix": [0, 6], "x": 6, "y": 0 },
+                { "matrix": [0, 7], "x": 7, "y": 0 },
+                { "matrix": [0, 8], "x": 8, "y": 0 },
+                { "matrix": [0, 9], "x": 9, "y": 0 },
+                { "matrix": [0, 10], "x": 10, "y": 0},
+                { "matrix": [0, 11], "x": 11, "y": 0},
+                { "matrix": [1, 0], "x": 0, "y": 1 },
+                { "matrix": [1, 1], "x": 1, "y": 1 },
+                { "matrix": [1, 2], "x": 2, "y": 1 },
+                { "matrix": [1, 3], "x": 3, "y": 1 },
+                { "matrix": [1, 4], "x": 4, "y": 1 },
+                { "matrix": [1, 5], "x": 5, "y": 1 },
+                { "matrix": [1, 6], "x": 6, "y": 1 },
+                { "matrix": [1, 7], "x": 7, "y": 1 },
+                { "matrix": [1, 8], "x": 8, "y": 1 },
+                { "matrix": [1, 9], "x": 9, "y": 1 },
+                { "matrix": [1, 10], "x": 10, "y": 1 },
+                { "matrix": [1, 11], "x": 11, "y": 1 },
+                { "matrix": [2, 0], "x": 0, "y": 2 },
+                { "matrix": [2, 1], "x": 1, "y": 2 },
+                { "matrix": [2, 2], "x": 2, "y": 2 },
+                { "matrix": [2, 3], "x": 3, "y": 2 },
+                { "matrix": [2, 4], "x": 4, "y": 2 },
+                { "matrix": [2, 5], "x": 5, "y": 2 },
+                { "matrix": [2, 6], "x": 6, "y": 2 },
+                { "matrix": [2, 7], "x": 7, "y": 2 },
+                { "matrix": [2, 8], "x": 8, "y": 2 },
+                { "matrix": [2, 9], "x": 9, "y": 2 },
+                { "matrix": [2, 10], "x": 10, "y": 2 },
+                { "matrix": [2, 11], "x": 11, "y": 2 }
+            ]
+        }
+    }
+}
```

Lint has some issue with the file. An extra line may help. 
```suggestion
}

```

---

Thanks